src/js/themes/microvideo.js

```diff
@@ -31,7 +31,7 @@ template.innerHTML = /*html*/`
     color: var(--_primary-color);
   }
 
-  [disabled] {
+  [disabled]:not(media-live-button) {
```

Comment:
NOTE: `media-live-button` is `disabled` whenever not playing at the live edge and will automatically update its default cursor to `not-allowed`, so excluding it from this styling so it doesn't have partial opacity under these conditions.